frontends/android/BitBoxApp/app/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
     implementation "android.arch.lifecycle:extensions:1.1.1"
     implementation "android.arch.lifecycle:viewmodel:1.1.1"
     implementation project(path: ':mobileserver')
-
+    implementation 'com.google.mlkit:barcode-scanning:17.2.0'
```

Comment:
Please add a comment that the qr-code scanning JS lib uses this as the QR code scanner.